app/contexts/media-context.tsx

```
@@ -491,14 +514,37 @@
+        audioRef.current.src = '';
+      }
+    };
+  }, []);
+
+  // Handle audio element errors
+  useEffect(() => {
+    const audio = audioRef.current;
+    if (!audio) return;
+
+    const handleAudioError = (e: ErrorEvent) => {
+      handleError(new Error('Audio playback error: ' + e.message));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)